cmd/cloudshell_open/project.go

```diff
@@ -43,8 +43,12 @@ func listProjects() ([]string, error) {
 	}
 	b = bytes.TrimSpace(b)
 	p := strings.Split(string(b), "\n")
-	sort.Strings(p)
-	return p, err
+	if len(p) == 1 && p[0] == "" {
+		return []string{}, err
+	} else {
+		sort.Strings(p)
+		return p, err
```

Comment:
both `return x, err` should be `return x, nil`.
not sure why `err` is down here..